CMakeLists.txt

```diff
@@ -394,6 +398,11 @@ else()
   list(PREPEND VIAMCPPSDK_GRPCXX_REFLECTION_LIBRARIES ${VIAMCPPSDK_GRPCXX_REFLECTION_LIB})
 endif()
 
+# These are the minimum versions of gRPC and protobuf for which remote plugins with buf are supported.
```

Comment:
I lifted this check into a new variable because we need this test in more than one place.